README.md

```diff
@@ -0,0 +1,87 @@
+Trigger Codemagic workflow
+==========================
+
+GitHub Action to trigger a workflow on Codemagic.
+
+Quick start
+-----------
+
+Add the following configuration to `.github/workflows/main.yml` to trigger Codemagic build on any push event.
+
+    on: push
+
+    jobs:
+      trigger-codemagic-build:
+        runs-on: ubuntu-latest
+        steps:
+          - name: Trigger Codemagic build
+            uses: codemagic-ci-cd/trigger-codemagic-workflow-action@v1.0.0
+            with:
+              app-id: <MY-APPLICATION-ID>
+              workflow-id: <MY-WORKFLOW-ID>
+              token: ${{ secrets.CODEMAGIC_API_TOKEN }}
+
+Required `with` arguments:
+
+| Argument      | Description              |
+|---------------|--------------------------|
+| `app-id`      | Codemagic application ID |
+| `workflow-id` | Codemagic workflow ID    |
+| `token`       | Codemagic API token      |
```

Comment:
do we need to explain where to find those values or perhaps link to our API docs?